### ISSUE TITLE
Feature Request - Client Area Message

### ISSUE THREAD

**Issue by Digiw0lf** [osTicket/osTicket#2983]
Hello,

I would like to see a message that you can enable which adds a box with some kind of image to the side of it. This message could be customized via Admin panel for announcements such as holidays. Where clients can still submit tickets but understand that we are on holiday or that a service outage is happening.

**Comment by ntozier** [comment]
So you mean like:
Admin panel -> Manage -> Pages -> Landing Page

**Comment by Digiw0lf** [comment]
@ntozier Yes but without having to edit the page, it would just add it's self to it using the admin page but would follow page to page at the top of each page
